tests/integration/api_v3/http/transactions.test.js

```diff
@@ -596,4 +595,40 @@ xdescribe('Transactions API', () => {
 			}
 		});
 	});
+
+	describe('Transactions ordered by index', () => {
+		it('returns 10 transactions ordered by index descending', async () => {
+			const response = await api.get(`${endpoint}?order=index:desc`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Array);
+			expect(response.data.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.length).toBeLessThanOrEqual(10);
+			response.data.forEach((transaction, i) => {
+				expect(transaction).toMap(transactionSchema);
+				if (i > 0) {
+					const prevTx = response.data[i];
+					const prevTxIndex = prevTx.index;
+					expect(prevTxIndex).toBeGreaterThanOrEqual(transaction.index);
+				}
+			});
```

Comment:
This check would fail if the response contains transactions from separate blocks. Please adjust accordingly.